pkg/bigqueryexporter/bigquery.go

```diff
@@ -52,10 +52,13 @@ func (client *Client) ExportData(ctx context.Context, dbClient *db.DB) error {
 
 func (client *Client) ExportReleaseTags(ctx context.Context, dbClient *db.DB) error {
 	rows := make([]models.ReleaseTag, 0)
-	query := client.Query(`
-		SELECT
-			ROW_NUMBER() OVER() id, *
-		FROM ` + "`ci_data.ReleaseTags`")
+	query := client.Query(`SELECT * FROM (SELECT ROW_NUMBER() OVER() id, * FROM ` + "`ci_data.ReleaseTags`) WHERE id > @lastID)")
```

Comment:
Could you clarify the select row_number like you did in the commit msg with a comment on each use here? Will be helpful for future maintenance I think.

Should we be using an order by here? It feels a little bit coincidental that this produces consistent results and if it didn't, thinks might get a little weird? If you're confident this is documented and safe then disregard my concern.